src/App.js

```diff
@@ -1,116 +1,151 @@
-import React, { Component } from 'react';
-import './App.css';
+import React from 'react';
 import 'react-widgets/dist/css/react-widgets.css';
-import Multiselect from 'react-widgets/lib/Multiselect'
-import { Navbar, Alignment, Slider, Button } from "@blueprintjs/core";
+import Multiselect from 'react-widgets/lib/Multiselect';
+import {Navbar, Alignment, Slider, Button} from '@blueprintjs/core';
 import Grid from '@material-ui/core/Grid';
-import Screen from './Screen'
+import Screen from './Screen';
+import {connect} from 'react-redux';
+import {setScale, setLanguages, setScreens, setUrl, resetState} from './actions';
+import PropTypes from 'prop-types';
 
 // Useful:
 // https://jquense.github.io/react-widgets/api/Multiselect/
 // https://blueprintjs.com/docs/#core
 // https://dev.to/tylermcginnis/a-comprehensive-guide-to-reactjs-in-2018--4nbc
-
-const SCREENS = [
+// https://www.valentinog.com/blog/react-redux-tutorial-beginners/
+// https://itnext.io/react-component-class-vs-stateless-component-e3797c7d23ab
+export const SCREENS = [
   {device: 'iPhone 7', os: 'iOS', width: 750, height: 1334},
   {device: 'iPhone 7+', os: 'iOS', width: 1080, height: 1920},
   {device: 'Samsung Galaxy S6', os: 'Android', width: 1440, height: 2560},
   {device: 'Google Pixel', os: 'Android', width: 1800, height: 1920},
   {device: 'iPhone 4', os: 'iOS', width: 640, height: 960},
   {device: 'Nokia Lumia', os: 'Windows', width: 480, height: 800},
   {device: 'iPhone 3G', os: 'iOS', width: 320, height: 480},
-].map(item => { return {ratio: (item.height/item.width).toFixed(2), ...item}})
+].map (item => {
+  return {ratio: (item.height / item.width).toFixed (2), ...item};
+});
```

Comment:
oh I love this way (array & map) of declaring constants :heart_eyes: 

p.s.  could be `item => ({ratio: (item.height / item.width).toFixed (2), ...item}}` - eslint should tell you that

Reply:
`toFixed (2)` - no space

Reply:
`prettier` does some weird stuff, like spaces